.travis.yml

```diff
@@ -21,53 +23,108 @@ addons:
 
 env:
     global:
+
         # The following versions are the 'default' for tests, unless
-        # overidden underneath. They are defined here in order to save having
+        # overridden underneath. They are defined here in order to save having
         # to repeat them for all configurations.
+        - PYTHON_VERSION=3.6
+        - NUMPY_VERSION=stable
```

Comment:
@nmearl - I think you can work around the current failures in Travis by setting NUMPY_VERSION=1.12 here instead of stable - the issue is that Astropy 1.0 isn't built with Numpy 1.13 in conda yet.